vendor/bat-native-ledger/src/bat_helper.cc

```diff
@@ -2269,7 +2269,7 @@ static bool ignore_ = false;
             if (iter != data.end()) {
               std::string idAddition(iter->second);
               if (idAddition.find('v') != std::string::npos) {
-                id += "_vod_" + braveledger_bat_helper::split(idAddition, 'v')[1];
+                id = "vod_" + braveledger_bat_helper::split(idAddition, 'v')[1];
```

Comment:
this shouldn't be changed as now we are not recording vod's correctly. This needs to be reverted

![image](https://user-images.githubusercontent.com/9574457/51109982-38ded780-17f7-11e9-9710-e35fc4551c79.png)